databricks/koalas/frame.py

```diff
@@ -4008,6 +4008,66 @@ def duplicated(self, subset=None, keep="first") -> "Series":
             )
         )
 
+    # TODO: support other as DataFrame or array-like
+    def dot(self, other: "Series") -> "Series":
+        """
+        Compute the matrix multiplication between the DataFrame and other.
+
+        This method computes the matrix product between the DataFrame and the
+        values of an other Series
+
+        It can also be called using ``self @ other`` in Python >= 3.5.
+
```

Comment:
Shall we add the same note as `transpose` as this uses it internally?

Reply:
Makes sense. Added it!